[2024-02-02 06:20:22,512] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: project_6_26.aggregation_test manual__2024-02-02T06:20:07.954489+00:00 [queued]>
[2024-02-02 06:20:22,526] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: project_6_26.aggregation_test manual__2024-02-02T06:20:07.954489+00:00 [queued]>
[2024-02-02 06:20:22,526] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2024-02-02 06:20:22,526] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2024-02-02 06:20:22,527] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2024-02-02 06:20:22,553] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): aggregation_test> on 2024-02-02 06:20:07.954489+00:00
[2024-02-02 06:20:22,560] {standard_task_runner.py:52} INFO - Started process 4685 to run task
[2024-02-02 06:20:22,563] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'project_6_26', 'aggregation_test', 'manual__2024-02-02T06:20:07.954489+00:00', '--job-id', '411', '--raw', '--subdir', 'DAGS_FOLDER/main.py', '--cfg-path', '/tmp/tmpamrm4gm_', '--error-file', '/tmp/tmp9ck7syiz']
[2024-02-02 06:20:22,564] {standard_task_runner.py:80} INFO - Job 411: Subtask aggregation_test
[2024-02-02 06:20:22,771] {task_command.py:369} INFO - Running <TaskInstance: project_6_26.aggregation_test manual__2024-02-02T06:20:07.954489+00:00 [running]> on host 20775c919683
[2024-02-02 06:20:23,003] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=project_6_26
AIRFLOW_CTX_TASK_ID=aggregation_test
AIRFLOW_CTX_EXECUTION_DATE=2024-02-02T06:20:07.954489+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-02-02T06:20:07.954489+00:00
[2024-02-02 06:20:23,112] {logging_mixin.py:115} INFO -    id  kode_prov  case_id     tanggal  total
0   1         32        1  2020-08-05   1815
[2024-02-02 06:20:23,132] {logging_mixin.py:115} INFO -    kode_prov  id  case_id  year  total
0         32  10        1  2020   2507
[2024-02-02 06:20:23,132] {python.py:173} INFO - Done. Returned value was: None
[2024-02-02 06:20:23,188] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=project_6_26, task_id=aggregation_test, execution_date=20240202T062007, start_date=20240202T062022, end_date=20240202T062023
[2024-02-02 06:20:23,234] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-02-02 06:20:23,403] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
